projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx

```diff
@@ -8,11 +12,46 @@ interface VideoPressValueSectionProps {
 }
 
 const VideoPressValueSection: FC< VideoPressValueSectionProps > = ( { isPluginActive, data } ) => {
-	if ( ! isPluginActive && data.videoCount ) {
-		return <span className="videopress-card__video-count">{ data.videoCount }</span>;
+	const { videoCount, featuredStats } = data || {};
+	const shortenedNumberConfig: Intl.NumberFormatOptions = {
+		maximumFractionDigits: 1,
+		notation: 'compact',
+	};
+
+	if ( ! videoCount ) {
```

Comment:
this check might need to ensure videoCount is an int and then check for `>0`. 
For some reason, I still saw the card with 0s for stats:

![Screenshot 2024-08-13 at 2 30 42 PM](https://github.com/user-attachments/assets/b44c9014-4bc8-484e-bf82-4d16c18cb731)

Maybe my video count was coming in as a string or perhaps typescript does not equate the integer value of 0 with boolean false?

Reply:
Ah ok, I think I misunderstood the discussion. I was thinking we wanted to hide the stats if they were 0, but I understand the rationale of showing them too, so fine to leave this IMO.

In the tooltip PR though, I do think we should edit the language that says "your video is live and getting views" if there are 0 views 😅

Reply:
That's fair 🤔 Maybe I'll update it to something like "Your video is live! It should start gathering views soon" or something like that. That's not great, I'm not great with copy 😅